index.html

```diff
@@ -33,10 +33,10 @@ <h1 class="jumbotron-heading">Projects List</h1>
                         <a href="credits.html" class="btn btn-primary">Main Project</a>
                     </p>
                     <p>
-                        <a href="#" class="btn btn-primary">Project 01</a>
+                        <a href="mandeep.html" class="btn btn-primary">Open Weather API</a>
                     </p>
                     <p>
-                        <a href="#" class="btn btn-primary">Project 02</a>
```

Comment:
Why are you using two buttons on the root index.html file?

Reply:
sorry i forget to remove it
i  will fix it